Gemfile.lock

```diff
@@ -536,4 +536,4 @@ DEPENDENCIES
   yabeda-sidekiq
 
 BUNDLED WITH
-   2.5.23
+   2.6.1
```

Comment:
Does this version aligns with what is in the rhel containers?

Reply:
We're [installing bundler ourselves on top](https://github.com/RedHatInsights/compliance-backend/blob/master/Dockerfile#L29).

This PR is essentially me doing manually the automation's work that was [buggy](https://github.com/RedHatInsights/compliance-backend/pull/2334). (this was fixed with the other PR)
I just did a `bundle update --bundler` inside the container.